frontend/src/components/Button.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+
+class Button extends Component {
+  render() {
+    return (
+      <div className="button">
+        <a href={this.props.buttonlink}>{this.props.button}</a>
```

Comment:
If you use `this.props.children` instead of this.props.button, you can use the button like this:
```javascript
<Button>Yes</Button>
```
https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891

Also this component would make an excellent function component :)

Reply:
implemented it for all relevant components. :)